Lib/test/test_configparser.py

```diff
@@ -691,7 +700,21 @@ def test_write(self):
                     "option-without-value\n"
                     "\n"
                     )
-            self.assertEqual(output.getvalue(), expect_string)
+            if testfile is None:
+                observed = output.getvalue()
+            else:
+                with open(testfile, 'r') as testf:
+                    observed = testf.read()
+            self.assertEqual(observed, expect_string)
+
+    def test_write_filename(self):
+        f = tempfile.NamedTemporaryFile(delete=False)
+        try:
+            self.test_write(testfile=f.name)
+        except:
+            raise
```

Comment:
I believe these two lines can be omitted.  `try; ... finally` without except is legal.